### ISSUE TITLE
Visualisations and trajectories

### ISSUE THREAD

**Issue by chrisbarnettster** [galaxycomputationalchemistry/galaxy-tools-compchem#33]
We need something to do this, otherwise it is easy to leave galaxy. 

An interactive env, tool? what are the options? 

@simonbray

**Comment by simonbray** [comment]
Need to develop NGLviewer further.

Some 2D viewer for chemical structures - something like this -> http://molview.org/

**Comment by chrisbarnettster** [comment]
Ok great! Also maybe something for docking to view the docking search space box when using autodock.

nglviewer - http://nglviewer.org/ngl/
molview - http://molview.org

**Comment by chrisbarnettster** [comment]
what about other formats? Current viewer should be able to read pdb, gro, xyz etc/

**Comment by chrisbarnettster** [comment]
@simonbray Dan Blankenberg's interactive envs also look like a great option. Can you think of any tools that are on docker - e.g. avogadro, autodock tools viewer or similar?

**Comment by bgruening** [comment]
http://nglviewer.org/ngl/api/manual/file-formats.html#sdf SDF should be supported. Is our version just too old?

**Comment by chrisbarnettster** [comment]
Yes it is supported. minor mod to Galaxy is needed. https://github.com/galaxyproject/galaxy/issues/8278
